pathsetup.sh.in

```diff
@@ -2,7 +2,7 @@
 #
 # Shell script for preparing the user's shell startup scripts for Fink.
 # Copyright (c) 2003-2005 Martin Costabel
-# Copyright (c) 2003-2016 The Fink Package Manager Team
+# Copyright (c) 2003-2019 The Fink Package Manager Team
```

Comment:
2020?

Reply:
The original change was 2019. I will fix it directly after merging this.